azure-pipelines.yml

```diff
@@ -11,26 +11,72 @@ pr:
       - develop
   drafts: false
```

Comment:
move this line after 7 for me please

Reply:
Sorry I do not understand, can you please explain again?